src/services/merchantWebhookQueue.service.ts

```
@@ -101,12 +108,29 @@ export class MerchantWebhookQueueService {
+              retryCount: attemptsMade,
+              
+            }
+          );
+
           console.log(
             `Webhook delivered successfully to ${merchantWebhook.url} after ${attemptsMade} attempt(s)`,
           );
 
           return { success: true };
-        } catch (error) {
+        } catch (error:any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)